ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKGroupedTableTest.java

```diff
@@ -174,7 +174,7 @@ public void init() {
 
     when(aggregateSchema.value()).thenReturn(mock(List.class));
 
-    when(mockKGroupedTable.aggregate(any(), any(), any(), any())).thenReturn(table);
+    when(mockKGroupedTable.aggregate(any(), any(), any(), (Materialized)any())).thenReturn(table);
```

Comment:
Would be cleaner to do `any(Materialized.class)`.

Reply:
Ack.  I updated one instance of `(Materialized)any()` to `any(Materialized.class)` on line 314.  

But changing this line to use `any(Materialized)` caused `NullPointerException`s in the test.  

The reason for this from what I can see is that this function returns `null` instead of a `Materialized` instance in the test:
 https://github.com/confluentinc/ksql/blob/master/ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedTable.java#L154-L158

By changing the mock to expect a `Materialized.class` explicitly I think the mock returns `null` instead of the `KTable` instance when this is called ( https://github.com/confluentinc/ksql/blob/master/ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedTable.java#L160-L164)

I'm thinking this is ok as is, but just let me know what you think.